src/pandas_profiling/model/describe.py

```diff
@@ -50,9 +49,6 @@ def describe(
             - package: package details.
     """
 
-    if df is None:
```

Comment:
is it in purpose or a wrong merge?

Reply:
Logic not needed as this method is not user interfaced. You already have this validation done earlier, too many duplicate and hard to maintain validation rules as they are spread across files.